samples/plugins/mysql_odbc/manifest.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version='1.0' encoding='utf-8' ?>
+
+<connector-plugin class='mysql_odbc_connector' superclass='mysql_odbc' plugin-version='0.0.0' name='MySQL ODBC Plugin Connector' version='18.1'>
+  <vendor-information>
+    <company name='TestCompany' />
+    <support-link url='http://google.com' />
```

Comment:
Don't use google as the support link, just have [support-link] or don't have the link there.

Reply:
Use Ross's suggestion for the link up above